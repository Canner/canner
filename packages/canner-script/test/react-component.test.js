/** @jsx builder */
import Sort from './toolbar/sort';

// eslint-disable-next-line
import builder from '../src/index';
import * as React from 'react';
import renderer from 'react-test-renderer';

describe('get react component from canner script', () => {
  it('should works', () => {
    const schema = <toolbar>
      <sort component={Sort} />
    </toolbar>;
    const component = renderer.create(React.createElement(schema.sort.component));
    expect(component.toJSON()).toMatchSnapshot();
  });
});


