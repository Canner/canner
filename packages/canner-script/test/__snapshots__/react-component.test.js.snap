// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get react component from canner script should works 1`] = `
<div>
  123
</div>
`;
